docs/developer/release/release_push.py

```
@@ -152,6 +152,16 @@ def stage_maven_artifacts_in_maven_central() -> None:
         ),
         working_dir=CHECKER_FRAMEWORK,
     )
+def close_maven_artifacts_in_maven_central() -> None:
```

Ruff (0.16.1)

[error] Expected 2 blank lines, found 0

Add missing blank line(s)

(E302)